docs/source-fabric/guide/checkpoint/distributed_checkpoint.rst

```diff
@@ -183,4 +183,36 @@ Note that you can load the distributed checkpoint even if the world size has cha
 Convert a distributed checkpoint
 ********************************
 
-Coming soon.
+It is possible to convert a distributed checkpoint to a regular, single-file checkpoint with this utility:
```

Comment:
This should warn that it's strictly for FSDP generated checkpoints.

Did you decide if this would eventually encapsulate deepspeed checkpoints too?

Reply:
The whole guide is centered around FSDP, I would prefer not to deviate from that and highlight FSDP as the primary strategy for model-parallel. Instead of a warning in the docs, I could make an error in the CLI that only FSDP checkpoints are supported. Would that be ok?

> Did you decide if this would eventually encapsulate deepspeed checkpoints too?

It's easy to add this into the utility to support both FSDP and DeepSpeed, since we can easily distinguish and detect the two checkpoint formats. I might do it in a follow up for consistency and completeness.

Reply:
> Instead of a warning in the docs, I could make an error in the CLI that only FSDP checkpoints are supported. Would that be ok?

Yes please!